src/components/Views/SyncView/DeviceCell.js

```diff
@@ -119,43 +129,35 @@ const DeviceCell = (props: Props) => {
         </View>
         {!syncInProgress && (
           <View style={styles.syncButtonContainer}>
-            <View style={styles.syncButtonOuterCircle}>
-              <View style={styles.syncButtonInnerCircle}>
-                <Icon
-                  color="white"
-                  name="arrow-forward"
-                  size={23}
-                  style={styles.syncArrow}
-                />
-              </View>
-            </View>
+            {syncIcon}
           </View>
         )}
         {syncInProgress && (
           <View style={styles.syncButtonContainer}>
```

Comment:
when sync is in progress, show the circles. we should refactor this once we get real progress events!